packages/ui/src/lib/dropdown/Dropdown.svelte

```diff
@@ -0,0 +1,210 @@
+<script lang="ts">
+import { faCaretDown, faCheck } from '@fortawesome/free-solid-svg-icons';
+import Fa from 'svelte-fa';
+
+export let id: string | undefined = undefined;
+export let name: string | undefined = undefined;
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export let value: any = '';
```

Comment:
IMHO you should use $props for new components as reactivity is better handled (it can say for example if we're missing some $state on some fields, etc)

Reply:
Updates to props. As per comment, this triggered requiring everything to Svelte 5.